### PR TITLE
brianr, all tests passing

### PR DESCRIPTION
**Pull request by brianr001**

### CODE REVIEW

**Comment by santiagobasulto** [comment]
This PR will be reviewed by: @cheeyim

**Review by cheeyim — CHANGES_REQUESTED** [review]
added some comments on minor style guide change.

File: dealership/vehicles.py [hunk 1]
```diff
@@ -1,15 +1,63 @@
+# things to try: make it DRYer by implementing sale_price and purchase_price
+# methods into Vehicle class, rather than repeating in each type of vehicle.
+
 class Vehicle(object):
+    
+    # sale multipliers 
+    
+    car_sale_mult = 1.2
```
Comment (cheeyim) [inline dealership/vehicles.py:8]: We can change these constants caps, following the PEP-8 styleguide - https://www.python.org/dev/peps/pep-0008/#constants

File: dealership/contracts.py [hunk 2]
```diff
@@ -1,12 +1,50 @@
...
+        if self.customer.is_employee():
+            self.discount = .90
+        else:
+            self.discount = 1
+        
+    
+    def total_value(self): # total value of contract paid by customer
+        
+        total = (self.vehicle.sale_price() + (self.vehicle.interest_rate * \
+        self.monthly_payments * self.vehicle.sale_price() / 100)) * self.discount
```
Comment (cheeyim) [inline dealership/contracts.py:21]: can we indent this for readability?

File: dealership/vehicles.py (hunk 1 above)
Comment (brianr001) [reply dealership/vehicles.py:8]: Good suggestion! I need to pay better attention to that stuff.

File: dealership/contracts.py (hunk 2 above)
Comment (brianr001) [reply dealership/contracts.py:21]: I'll try that to see if it works...I bet it will.

**Pull request closed without merging**